tests/cephfs/custom_fscid_cephfs.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        """
+        Pre-requisites :
+        1. creats fs volume create cephfs if the volume is not there
+
+        Custom fscid cephfs operation:
+        1. Delete existing cephfs
+        ceph fs volume rm <cephfs_name> --yes-i-really-mean-it
+        Ex. ceph fs volume rm cephfs --yes-i-really-mean-it
+        2. Create data & metadata pools for cephfs
+        Ex. ceph osd pool create cephfs_data 64 64
+            ceph osd pool create cephfs_metadata 8 8
+        3. Create Cephfs with data & metadata pool with specific fscid
+        ceph fs new <cephfs_name> <metada_pool> <data_pool> --fscid <fscid> --force
+        Ex. ceph fs new cephfs cephfs_metadata cephfs_data --fscid {fscid} --force
+        4. Verify cephfs is created with specified fscid
+        ceph fs ls
+        ceph fs dump
+        """
+        tc = "CEPH-83574632"
+        log.info("Running cephfs %s test case" % (tc))
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        clients = ceph_cluster.get_ceph_objects("client")
+        build = config.get("build", config.get("rhbuild"))
+        rhbuild = config.get("rhbuild")
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        client1 = clients[0]
+        fscid = "300"
+        commands = [
+            "ceph config set mon mon_allow_pool_delete true",
+            "ceph fs volume rm cephfs --yes-i-really-mean-it",
+        ]
+        for command in commands:
+            client1.exec_command(sudo=True, cmd=command)
+        verifyremove_command = [
+            "ceph fs ls | grep cephfs",
+            "ceph osd lspools | grep cephfs.cephfs",
+        ]
+        for command in verifyremove_command:
+            client1.exec_command(sudo=True, cmd=command, check_ec=False)
+            if client1.node.exit_status == 1:
+                log.info(f"{command} successfully executed")
+        commands = [
+            "ceph osd pool create cephfs_data 64 64",
+            "ceph osd pool create cephfs_metadata 8 8",
+            f"ceph fs new cephfs cephfs_metadata cephfs_data --fscid {fscid} --force",
```

Comment:
Hi Yogesh,
Can we create new file system with fscid. instead of deleting the existing and recreating it

Reply:
Done Amar